package/package.json

```diff
@@ -144,8 +144,8 @@
     "react": "18.2.0",
     "react-native": "0.73.6",
     "react-native-builder-bob": "0.23.1",
-    "react-native-gesture-handler": "~2.16.1",
-    "react-native-reanimated": "~3.10.0",
+    "react-native-gesture-handler": "~2.21.2",
```

Comment:
Note: We should probably update the docs as well to reflect this, as we have some versions mentioned there as well. Nothing urgent, just something to keep in mind.